phpunit.xml.dist

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<phpunit bootstrap="./vendor/autoload.php" colors="true">
+<phpunit colors="true">
```

Comment:
Is this now automatically included?

Reply:
I don't recall a time when it was not. PHPUnit should be launched with `bin/phpunit`. If you invoke it correctly, it will always include the autoloader. Perhaps if you're one of these people that thinks installing PHPUnit as a global PHAR is a good idea then it might not, but even that is probably no longer the case these days.